### PR TITLE
Move damage colors to global SCSS variables

### PR DESCRIPTION
**Pull request by In3luki**

### CODE REVIEW

File: src/styles/_colors.scss [hunk 1]
```diff
@@ -29,6 +29,26 @@ $tertiary-light-color: lighten($tertiary-color, 10) !default;
...
+$damage-force: #6300aa;
+$damage-good: #9d730a;
+$damage-lawful: #402600;
+$damage-mental: midnightblue;
+$damage-negative: #00001f;
+$damage-piercing: $damage-physical;
+$damage-poison: #5b7332;
+$damage-positive: #ffffe0;
+$damage-slashing: $damage-physical;
+$damage-sonic: darkcyan;
```
Comment (stwlam) [inline src/styles/_colors.scss:51]: Make them CSS variables if you're going to do that

Comment (In3luki) [reply src/styles/_colors.scss:51]: I've added CSS variables based on the SCSS variables. Apparently functions like `color.adjust` don't work with CSS variables.

Comment (stwlam) [reply src/styles/_colors.scss:51]: Ohhh, right, right.

Comment (stwlam) [reply src/styles/_colors.scss:51]: Ehh, let's nix that, then, sorry. All it'll do if anyone tries to override them is not work for some reason.

File: src/styles/_colors.scss [hunk 2]
```diff
@@ -105,4 +125,25 @@ $adjusted-lower: #cc3311 !default;
...
+  --damage-good: $damage-good;
+  --damage-lawful: $damage-lawful;
+  --damage-mental: $damage-mental;
+  --damage-negative: $damage-negative;
+  --damage-physical: $damage-physical;
+  --damage-piercing: $damage-piercing;
+  --damage-poison: $damage-posion;
+  --damage-positive: $damage-positive;
+  --damage-slashing: $damage-slashing;
+  --damage-sonic: $damage-sonic;
```
Comment (stwlam) [inline src/styles/_colors.scss:148]:
```suggestion
```

**Pull request merged**